src/Sculpin/Bundle/TwigBundle/FilesystemLoader.php

```diff
@@ -23,7 +23,7 @@ class FilesystemLoader extends \Twig_Loader_Filesystem
     /**
      * {@inheritdoc}
      */
-    public function getCacheKey($name)
+    public function getCacheKey($name): string
```

Comment:
The `$name` type is missed.

Reply:
It will not work, because there will be a mismatch of the method declaration.

Reply:
Is the type of `$name` `mixed`?

Reply:
In the original Twig class, the type is simply not specified, so it cannot be specified in the child class.